<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[It's gone!]]>
    </line>
    <line>
      <![CDATA[It's back!]]>
    </line>
  </reporter-output>
  <suite started-at="2025-03-13T13:49:31 EDT" name="Default suite" finished-at="2025-03-13T13:49:39 EDT" duration-ms="8413">
    <groups>
    </groups>
    <test started-at="2025-03-13T13:49:31 EDT" name="Default test" finished-at="2025-03-13T13:49:39 EDT" duration-ms="8413">
      <class name="scripts.Hero">
        <test-method is-config="true" signature="openapp()[pri:0, instance:scripts.Hero@470734c3]" started-at="2025-03-13T13:49:28 EDT" name="openapp" finished-at="2025-03-13T13:49:31 EDT" duration-ms="2457" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openapp -->
        <test-method signature="home()[pri:0, instance:scripts.Hero@470734c3]" started-at="2025-03-13T13:49:31 EDT" name="home" finished-at="2025-03-13T13:49:38 EDT" duration-ms="7571" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [It's back!I] but found [It's back!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [It's back!I] but found [It's back!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at pages.HeroPage.homie(HeroPage.java:52)
at scripts.Hero.home(Hero.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[It's gone!]]>
            </line>
            <line>
              <![CDATA[It's back!]]>
            </line>
          </reporter-output>
        </test-method> <!-- home -->
        <test-method is-config="true" signature="closeapp()[pri:0, instance:scripts.Hero@470734c3]" started-at="2025-03-13T13:49:38 EDT" name="closeapp" finished-at="2025-03-13T13:49:39 EDT" duration-ms="789" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeapp -->
      </class> <!-- scripts.Hero -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
